example/spec/parallel/syncGranule/SyncGranuleDuplicateHandlingSpec.js

```diff
@@ -204,6 +209,15 @@ describe('When the Sync Granule workflow is configured', () => {
       });
 
       it('captures both files', async () => {
+        // This assertion is to check that the granule has been updated in dynamo
+        // before performing further checks
+        const record = await waitForModelStatus(
```

Comment:
I think we want to do the same thing elsewhere in this spec where it checks the granule after an execution:

https://github.com/nasa/cumulus/blob/master/example/spec/parallel/syncGranule/SyncGranuleDuplicateHandlingSpec.js#L251